### PR TITLE
Less agressive grayscale/opacity in SelectionList

### PR DESCRIPTION
**Pull request by jacogr**
@ngotchac Pulls in the ideas from your fix as well. Could you check that 

(a) the consistency between selected/unselected is right
(b) the consistency between default/select is right

I'm thinking it probably still needs some small tweaks.

### CODE REVIEW

File: js/src/ui/SelectionList/selectionList.css [hunk 1]
```diff
@@ -16,22 +16,25 @@
...
   flex: 1;
   height: 100%;
   position: relative;
   width: 100%;
 
   &:hover {
-    box-shadow: inset 0 0 0 2px rgb(255, 255, 255);
+    border-color: transparent;
+    filter: none;
+    opacity: 1;
```
Comment (ngotchac) [inline js/src/ui/SelectionList/selectionList.css]: Isn't `filter` and `opacity` only needed for the `.unselected` class ?

File: js/src/ui/SelectionList/selectionList.css [hunk 2]
```diff
@@ -43,13 +46,17 @@
...
+
+  &.default {
+    border-color: rgba(255, 255, 255, 0.75);
+  }
 }
 
 .unselected {
-  filter: grayscale(100%);
-  opacity: 0.5;
+  filter: grayscale(10%);
```
Comment (ngotchac) [inline js/src/ui/SelectionList/selectionList.css]: I liked it a bit more gray actually... Could increase this a bit

File: js/src/ui/SelectionList/selectionList.css (hunk 1 above)
Comment (jacogr) [inline js/src/ui/SelectionList/selectionList.css]: This just resets everything. Could be specific, i.e. `&.unlselected:hover { ... }` - simpler this way, no impact either way.

File: js/src/ui/SelectionList/selectionList.css (hunk 2 above)
Comment (jacogr) [inline js/src/ui/SelectionList/selectionList.css]: Me too, actually. Had to contain it a bit to not look totally disabled, yet, but more subtle.

**Pull request merged**